src/disabled-tasks/collapsible-list.tsx

```diff
@@ -0,0 +1,330 @@
+/**
```

Comment:
Adding this component temporarily until we release this [PR](https://github.com/woocommerce/woocommerce/pull/36806) in the core